src/Components/Card.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const Card = ({card, isFlipped, onClick}) => {
+
+    return (
+        <div className={isFlipped? "card flipped" : "card"} onClick={onClick}>
```

Comment:
It is better to use [clsx](https://www.npmjs.com/package/clsx) library to combine classes dynamically.